### PR TITLE
Doc update

### LINKED ISSUE #588: Update Docs
- [x] New workflow formats
- [x] Changes to simput within a workflow
- [x] Other updates

### PR DESCRIPTION
**Pull request by TristanWright**
closes #588

### CODE REVIEW

**Comment by TristanWright** [comment]
this is ready.

**Comment by cjh1** [comment]
This looks great to me, I will let @jourdain review the workflow sections as he is more familiar with it.

File: docs/content/docs/workflows__pages.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+```sh
+src/workflows/generic/components/steps
+```
+
+## Input
+
+Import the component `generic/components/steps/SimputReact` and pass the following props: 
+
+- `simputType`, the name of the simput type
+- `inputFileKeys`, arguments to simput on the key for the file in the simulation metadata `key: String`, the name of the file `name: String`, and if the input file should be parsed `parse: Boolean`
```
Comment (jourdain) [inline docs/content/docs/workflows__pages.md]: Those are not necessary for simput. This list the files that either come in your project and/or simulation page that should be considered as Input and pass along the flow.

File: docs/content/docs/workflows__pages.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- `initialDataModel`, the initial data model for simput. Of format:  
+  ```js
+    {
+      data: {},
+      type: `${simput_type_name}`,
+      hideViews: [], // strings of the view names you would like to hide
+    }
+  ```
+
+- `simputModelDecorator`, a function which takes model and props
```
Comment (jourdain) [inline docs/content/docs/workflows__pages.md]: And allow to expand the actual data model.

**Comment by jourdain** [comment]
Looks good otherwise.

**Comment by TristanWright** [comment]
@jourdain made suggested changes

**Comment by TristanWright** [comment]
@jourdain, sorry need to approve to merge

**Pull request merged**